selfdrive/ui/qt/onroad.cc

```diff
@@ -75,6 +77,13 @@ void OnroadWindow::updateState(const UIState &s) {
     bg = bgColor;
     update();
   }
+
+  // force paint now
+  double tsm = (float)(nanos_since_boot() - sm["modelV2"].getLogMonoTime()) / 1e6;
```

Comment:
This line may have problems in replay mode.

Reply:
we want the best timestamp for when the model actually ran, so needs to be logMonoTime. I added another check to handle replay